test/jake/jakelib/migrate-test.js

```diff
@@ -717,6 +717,40 @@ test('migrate runs "esm" modules', (temp) => {
   });
 });
 
+test('migrate:unlock restores lock table to one row with is_locked=0', (temp) => {
+  const db = new sqlite3.Database(`${temp}/db`);
+  // Seed multiple rows into the lock table
+  return new Promise((resolve, reject) =>
+    db.run('INSERT INTO knex_migrations_lock(is_locked) VALUES (?),(?),(?)', [1, 0, 1], (err) => {
+      err ? reject(err) : resolve();
+    })
+  )
+    .then(() => {
+      return assertExec(
```

Comment:
Would be awesome if you could use new style of executing script and asserting results - see https://github.com/knex/knex/blob/master/test/cli/migrate.spec.js for an example

Reply:
Ah okay, is that the desired directory for net-new CLI tests? If so I will move my test over into that file.

Reply:
yes, we use testlab-cli instead of plain jake now.

Reply:
Sounds good! The new format is much cleaner and more familiar, so I'm all for switching over.